### ISSUE TITLE
Alternative setup for cloud source repository

### ISSUE THREAD

**Issue by jclee2** [terraform-google-modules/terraform-example-foundation#1299]
### TL;DR

Alternative and/or patch to 0-bootstrap now that Cloud Source Repositories is deprecated for a new org setup? The setup fails now at the cicd project setup? Is there a workaround I'm missing? Is it out of scope to sub in github, gitlab, bitbucket, etc as the source control and still use cloud build?

➜  0-bootstrap git:(master) ✗ terraform apply bootstrap.tfplan
module.tf_source.module.cloudbuild_project.module.project-factory.module.project_services.google_project_service.project_services["sourcerepo.googleapis.com"]: Creating...
╷
│ Error: Request `Enable Project Service "sourcerepo.googleapis.com" for project "prj-b-cicd-aa85"` returned error: failed to enable services: failed on request preconditions: googleapi: Error 403: Permission denied to enable service [sourcerepo.googleapis.com]

**Comment by eeaton** [comment]
I'll close this as a duplicate, this is already being tracked in #1249.

The other deployment paths (github etc) still work while we're assessing how to replace CSR and what will be the new default path, however as written the deployment guidance there uses different CI tool (Github as repo uses Github Actions as the CICD tool, etc)
